bin/add_attachment.py

```diff
@@ -40,86 +40,118 @@
 import lxml.etree as ET
 import urllib.request
 
-ALLOWED_TYPES = ['pdf', 'pptx', 'zip']
-ATTACHMENT_TYPES = 'Poster Presentation Note Software Supplementary Dataset'.split()
+ALLOWED_TYPES = ["pdf", "pptx", "zip"]
+ATTACHMENT_TYPES = "Poster Presentation Note Software Supplementary Dataset".split()
+
 
```

Comment:
[PEP8 doesn't specify](https://www.python.org/dev/peps/pep-0008/#id25) whether to use single or double quotes. We could save a lot of commit history by allowing either.